crates/README.md

```diff
@@ -0,0 +1,12 @@
+# Rust crates for gRPC interfaces
+
+This is composed of 3 crates:
+
+1. `mc-probufs-tonic`: A crate providing a
```

Comment:
I'm up for suggestions on these crate names, I had `mc-grpc-` but it looked kind of weird for `mc-grpc-grpcio`.  `mc-proto-` or `mc-protobufs-` also seem reasonable.

Reply:
naming is being addressed from initial review